<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>430</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>50</red>
        <green>205</green>
        <blue>50</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>42</red>
        <green>46</green>
        <blue>51</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>25</red>
        <green>29</green>
        <blue>33</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>25</red>
        <green>29</green>
        <blue>33</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>50</red>
        <green>205</green>
        <blue>50</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>85</green>
        <blue>85</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>50</red>
        <green>205</green>
        <blue>50</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>21</red>
        <green>24</green>
        <blue>27</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>25</red>
        <green>29</green>
        <blue>33</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Highlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>56</red>
        <green>142</green>
        <blue>60</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>35</red>
        <green>39</green>
        <blue>43</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>50</red>
        <green>205</green>
        <blue>50</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>42</red>
        <green>46</green>
        <blue>51</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>25</red>
        <green>29</green>
        <blue>33</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>25</red>
        <green>29</green>
        <blue>33</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>50</red>
        <green>205</green>
        <blue>50</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>85</green>
        <blue>85</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>50</red>
        <green>205</green>
        <blue>50</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>21</red>
        <green>24</green>
        <blue>27</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>25</red>
        <green>29</green>
        <blue>33</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Highlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>56</red>
        <green>142</green>
        <blue>60</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>35</red>
        <green>39</green>
        <blue>43</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>42</red>
        <green>46</green>
        <blue>51</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>25</red>
        <green>29</green>
        <blue>33</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>25</red>
        <green>29</green>
        <blue>33</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>85</green>
        <blue>85</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>25</red>
        <green>29</green>
        <blue>33</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>25</red>
        <green>29</green>
        <blue>33</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>35</red>
        <green>39</green>
        <blue>43</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>QuickFileHosting</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LayoutDirection::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>631</width>
      <height>371</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="comboBox">
         <property name="maximumSize">
          <size>
           <width>110</width>
           <height>22</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgba(42, 46, 51, 1);</string>
         </property>
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Delete</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Recover</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Set Password</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Remove Password</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>22</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgba(42, 46, 51, 1);</string>
         </property>
         <property name="text">
          <string>Upload File</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="uploadInfoLabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="uploadProgressBar">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="mainWindowAccInfo">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QTableWidget" name="tableWidget">
       <property name="styleSheet">
        <string notr="true">QHeaderView::section{
	background-color: rgba(25, 29, 33,1);
}

QTableView QTableCornerButton::section{
	background-color: rgba(25, 29, 33,1);
}</string>
       </property>
       <property name="showGrid">
        <bool>true</bool>
       </property>
       <property name="columnCount">
        <number>5</number>
       </property>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>21</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>25</red>
         <green>29</green>
         <blue>33</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>127</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>25</red>
         <green>29</green>
         <blue>33</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>25</red>
         <green>29</green>
         <blue>33</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>25</red>
         <green>29</green>
         <blue>33</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>127</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>25</red>
         <green>29</green>
         <blue>33</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>25</red>
         <green>29</green>
         <blue>33</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>25</red>
         <green>29</green>
         <blue>33</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>127</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>25</red>
         <green>29</green>
         <blue>33</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>25</red>
         <green>29</green>
         <blue>33</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="styleSheet">
    <string notr="true">QMenuBar::item:selected { 
     background-color: rgba(40, 47, 53, 1);
}

QMenuBar {
	background-color: rgba(25, 29, 33,1);
}</string>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="styleSheet">
     <string notr="true">QMenu {
	background-color: rgba(25, 29, 33,1);
}</string>
    </property>
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionAccount"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <action name="actionAccount">
   <property name="text">
    <string>Account</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
